test/langtools/tools/javac/records/RecordCompilationTests.java

```diff
@@ -145,8 +145,10 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
         }
     }
 
+    boolean useAP;
+
     public RecordCompilationTests() {
-        boolean useAP = System.getProperty("useAP") == null ? false : System.getProperty("useAP").equals("true");
+        useAP = System.getProperty("useAP", "false").equals("true");
```

Comment:
Can't `Boolean.getBoolean("useAP")` be used here instead?

Reply:
it could be used, but the test could potentially be executed from command line too and there is no guarantee that the system property will be there if executed from the command line

Reply:
As I understand, call to `Boolean.getBoolean("useAP")` is identical to your variant as it also relies on `System#getProperty(String)` so I cannot see any different behaviour if using it:
https://github.com/openjdk/jdk/blob/8df3e72cea9217a7a10e2ac4bf4d1501a7fad43c/src/java.base/share/classes/java/lang/Boolean.java#L272-L279